api/v1beta1/diagnostic.proto

```diff
@@ -19,13 +19,15 @@ message GetMetricsResponse {
 	uint64 LostWrCount = 7; 
 	uint64 LostNtCapCount = 8; 
 	uint64 LostBPFLogsCount = 9;
-	repeated EventCount BPFPerfEventSubmitAttempts = 10;
-	repeated EventCount BPFPerfEventSubmitFailures = 11;
+	repeated BPFPerfEventStats BPFPerfEventStats = 10;
```

Comment:
does it make sense to drop `Perf` and name it `BPFEventStats` so if we move to a ring buffer, we could still use the same structure for representing attempts/failures?